akka-persistence-typed/src/main/mima-filters/2.9.5.backwards.excludes/32526-replicated-interceptor.excludes

```diff
@@ -0,0 +1,2 @@
+# new method
```

Comment:
always confusing when starting to branch off, maybe this should be 2.9.6 and a copy in 2.8.x ?

Reply:
Just pushed, didn't think I needed to put it elsewhere than 2.9.6. Why is the copy needed in a release branch?